dockers/docker-snmp/snmpd.conf.j2

```diff
@@ -52,7 +80,10 @@ rocommunity6 {{ snmp_rocommunity }}
 #  Note that setting these values here, results in the corresponding MIB objects being 'read-only'
 #  See snmpd.conf(5) for more details
 sysLocation    {{ snmp_location }}
-sysContact     Azure Cloud Switch vteam <linuxnetdev@microsoft.com>
+{% if SNMP_CONTACT is defined %}
+sysContact     {{ SNMP.CONTACT.keys()[0] }} {{ SNMP.CONTACT.values()[0] }}
```

Comment:
sysContact is only taken from config_db in this case, so if config_db is not used and yml is used, sysContact will not have any string. we can add previous sysContact string in else condition here.

Reply:
added the default contact we already have in the "else" clause.